src/lips.js

```diff
@@ -3833,7 +3833,7 @@ function transform_syntax(options = {}) {
     function log(x) {
         /* istanbul ignore next */
         if (is_debug()) {
-            console.log(x);
+            console.log(x); // @jcubic can you enter what this console log returns here
```

Comment:
This is yet another log function that is local to `transform_syntax` that is part of hygienic macros `syntax-rules`.

Reply:
Should be deleted now. I made a new commit.